iip-template.md

```diff
@@ -0,0 +1,100 @@
+# IIP Guide
+
+## Overview
+
+This guide outlines the structure and key elements of an IIP document, providing a template for authors to use when submitting proposals. Each IIP is identified by a unique number and follows a specific file naming convention.
+
+### File Naming Convention
+
+The file name for an IIP should adhere to the following convention:
+- The file is named with two digits, where a single-digit number is always prepended with a 0.
+- Examples:
+  - 01.md
```

Comment:
that's good to consider to keep file name like: `iip-xx.md`

Reply:
Yes, I will fix that